app/models/bike.rb

```diff
@@ -1,7 +1,10 @@
 class Bike < ApplicationRecord
-  validates_presence_of    :identifier
-  validates_uniqueness_of  :identifier
-  
-  belongs_to :current_station, class_name: :Station, foreign_key: :current_station_id, optional: true
+    validates_presence_of   :identifier,
+                            :bike_model,
+                            :charge,
```

Comment:
I am not sure we need charge as it will be a static option in HTML, and probably condition won't be needed either, unless I am misunderstanding their usage.

Reply:
Yes, I agree with these observations. I guess the necessary bike columns is the `id` and the `current_station` so I will change accordingly.